api/tests/unit/features/test_unit_features_views.py

```diff
@@ -538,3 +539,34 @@ def test_project_admin_can_create_mv_options_when_creating_feature(client, proje
 
     response_json = response.json()
     assert len(response_json["multivariate_options"]) == 1
+
+
+@pytest.mark.parametrize(
+    "client", [(lazy_fixture("master_api_key_client")), (lazy_fixture("admin_client"))]
```

Comment:
If I understand correctly, I think the parentheses around the lazy_fixture calls are obsolete? You can probably rewrite this as:

```python
@pytest.mark.parametrize(
    "client", [lazy_fixture("master_api_key_client"), lazy_fixture("admin_client")]
)
```